examples/plot_connectivity.py

```diff
@@ -50,19 +50,21 @@
 
 ###############################################################################
 # We can modify the connectivity list to test the effect of different
-# connectivity patterns. For example, we can remove all layer 2 inhibitory
-# connections. In the default network, the src_gids of each connection are
-# all the same cell type.. Connections are stored under ``conn['gid_pairs']``
-# as a dictionary indexed by src_gid:
-# ``{src_gid1: [target_gid1, target_gid2], ...]``. Each src_gid indexes a
-# list with its target gids
-new_connectivity = [conn for conn in net.connectivity
-                    if conn['src_type'] != 'L2_basket']
-net.connectivity = new_connectivity
+# connectivity patterns. For example, we can remove 90% all layer 2 inhibitory
+# connections.
+# This can be achieved with ``net.connectivity[idx].drop``, a function that is
+# available for each element of ``net.connectivity``. We can also visualize
+# any connection using ``net.connectivity.plot``.
+# Note that in the default network, the src_gids of each connection are
+# all the same cell type allowing for easy modifications.
+for conn in net.connectivity:
+    if conn['src_type'] == 'L2_basket':
+        conn.drop(0.1)
```

Comment:
`connectivity` is a private class. You shouldn't expose it (yet) ... could you perhaps expose `probability` using `add_connection` ? This feels like you're trying to modify the connection but we're still trying to settle on API for adding a connection